frontend/packages/kubevirt-plugin/src/components/modals/vm-status-modal/vm-status-modal.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { Modal } from '@patternfly/react-core';
+import { HandlePromiseProps, withHandlePromise } from '@console/internal/components/utils';
+import { ModalComponentProps } from '@console/internal/components/factory';
+import { ModalFooter } from '../modal/modal-footer';
+import { PAUSED_VM_MODAL_MESSAGE } from '../../../constants/vm';
+import { VMIKind } from '../../../types';
+import { unpauseVM } from '../../../k8s/requests/vmi/actions';
+
+const modalTitle = 'Edit pause state';
+
+const VMStatusModalComponent = ({
```

Comment:
```suggestion
export const VMStatusModal = withHandlePromise<VMStatusModalProps>(({
```

and then you dont have to specify that the props are `VMStatusModalProps` a few lines below

Reply:
Done!